docs/main.js

```diff
@@ -1,4 +1,4 @@
-import twitter from 'twitter-text';
+// import twitter from 'twitter-text';
```

Comment:
Let's just remove these instead of commenting them out.  We have no need for twitter functionality.